### PR TITLE
Mat 2989

### PR DESCRIPTION
**Pull request by adongare**
MAT-2989 Update Approved User Email - QDM
MAT-2988 Update 'Approved' Checkbox display in Admin Screen - QDM

Screen Shot 2021-05-24 at 5 31 42 PM

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] This PR is into the correct branch.
- [ ] JIRA ticket for this PR:
- [ ] JIRA ticket links to this PR
- [ ] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [ ] Test fixtures updated and documented as necessary ( see [internal wiki](https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) )
- [ ] Code coverage has not gone down and all code touched or added is covered. 
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here: 
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here: 
- [ ] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [ ] JIRA test links:
- [ ] Justification for using JIRA tests:
- [ ] JIRA tests have been added to sprint

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [ ] JIRA tests have been run and pass
- [ ] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [ ] JIRA tests have been run and pass
- [ ] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

File: app/assets/javascripts/templates/users/user_edit_dialog.hbs [hunk 1]
```diff
@@ -5,14 +5,6 @@
         <h1 class="modal-title" id="userEditModalTitle">Edit User</h1>
       </div>
       <div class="modal-body">
-{{!--         <div class="form-group"> --}}
-{{!--           <label for="first_name">First name:</label> --}}
-{{!--           <input type="text" class="form-control" id="first_name" name="first_name" aria-describedby="emailHelp" placeholder="Enter first name"> --}}
```
Comment (ghost) [inline app/assets/javascripts/templates/users/user_edit_dialog.hbs:10]: Oh, what happened to first/last name? 
it was a part of the prev stories

File: app/assets/javascripts/views/users_view.js.coffee [hunk 2]
```diff
@@ -58,8 +58,8 @@ class Thorax.Views.User extends Thorax.Views.BonnieView
     view = this
     changed = @model.changedAttributes()
     prevAttributes = _.pick(@model.previousAttributes(), _.keys(changed))
-    approveChanged = @model.hasChanged('approved')
-    approved = changed? && @model.changed.approved
+    approveChanged = @model.hasChanged('harp_id') && !(prevAttributes.harp_id && @model.changed.harp_id)
```
Comment (ghost) [inline app/assets/javascripts/views/users_view.js.coffee:61]: Oh, interesting. I was thinking we should re-factor activate/deactivate methods on the server-side with this story.

File: app/assets/javascripts/templates/users/user_edit_dialog.hbs (hunk 1 above)
Comment (adongare) [reply app/assets/javascripts/templates/users/user_edit_dialog.hbs:10]: this is how it looks. I thought it's not needed that's why commented. I removed unused code. 
Screen Shot 2021-05-24 at 5 33 02 PM

Comment (adongare) [reply app/assets/javascripts/templates/users/user_edit_dialog.hbs:10]: Should it be there?

File: app/assets/javascripts/views/users_view.js.coffee (hunk 2 above)
Comment (adongare) [reply app/assets/javascripts/views/users_view.js.coffee:61]: if we are continuing with approval flag, I think we can continue with existing actions(approve/disable )

**Review by ghost — APPROVED** [review]

**Pull request merged**